apps/web/src/components/HeroText.module.css

```
@@ -0,0 +1,142 @@
+    linear-gradient(90deg, #000 1.5px, transparent 1.5px);
+  -webkit-mask-size:
+    100% 100%,
+    60px 60px,
+    60px 60px;
+  -webkit-mask-composite: source-in, source-over, source-over;
+  /* Light needs more punch (colour on near-white); dark shows the lines readily. */
+  opacity: 0.4;
+  animation: hero-streaks-in 1.4s 0.2s ease-out both;
+}
+
+:global(.dark) .streaks {
```

Stylelint (17.12.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)